CONTRIBUTING.adoc

```diff
@@ -65,16 +65,16 @@ caches have not been configured correctly.
 Accepting the configuration settings as outlined above should be sufficient to avoid this, however if your caches are still broken, you may try to append the following lines to `/etc/nix/nix.conf` and/or `~/.config/nix/nix.conf`:
 
 ----
-substituters = https://hydra.iohk.io https://cache.nixos.org/
+substituters = https://cache.iog.io https://cache.nixos.org/
 trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
 extra-experimental-features = nix-command flakes
 ----
 
 On NixOS, set the following NixOS options:
 ----
 nix = {
-  binaryCaches          = [ "https://hydra.iohk.io" "https://cache.iog.io" ];
-  binaryCachePublicKeys = [ "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" ];
+  binaryCaches          = [ "https://cache.iog.io" "https://cache.nixos.org"];
+  binaryCachePublicKeys = [ "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=" "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" ];
```

Comment:
I think you should also update trusted key here: remove one of hydra but add one for cache.iog.io.

cache.iog.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= is my best known value of it

Reply:
I wasn't aware we had a signing key for `cache.iog.io`? We probably want to keep the hydra one indefinitely: that cache probably has a bunch of stuff that was built by hydra, I don't know if it has been re-signed by `cache.iog.io`?

Reply:
I'm sorry. Seems my comment was invalid.

Reply:
Does what you wrote work?